Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: build run repo install
+
```

Comment:
This file is not needed. You can test the builds locally easily using 
```
flatpak-builder --install app org.clawsmail.Claws-Mail.json --user --force-clean
```
Or by sending a MR to the repo once created & the bot will launch a build automatically.